be/src/exprs/time_functions.cpp

```diff
@@ -2106,4 +2106,41 @@ std::string TimeFunctions::info_reported_by_time_slice = "time used with time_sl
 #undef DEFINE_TIME_STRING_UNARY_FN
 #undef DEFINE_TIME_UNARY_FN_EXTEND
 
+int weekday_from_dow_abbreviation(std::string dow) {
```

Comment:
Actually, we can define this function as 
```suggestion
static int weekday_from_dow_abbreviation(const std::string& dow) {
```
1. `static` is because this function is only used in this file
2. const reference to reduce unnecessary copy, which can be more efficient.

Reply:
Great suggestion! done.